docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -0,0 +1,71 @@
+```markdown
+
+```
+
+# Step 6 - Enabling account registration
+
+We are trying to let our friends write their bike trips directly into our website. We would like to let them create Rides. As an Administrator of the website you could create the Users yourself but In order to let them create their own content, first, you want to let them create their own accounts.
```

Comment:
Maybe "upload to our website" or "place on our website" instead of "write"?
Also,  "In order" should not be capitalized.